app/src/main/java/de/qspool/clementineremote/backend/player/MyLibrary.java

```diff
@@ -61,6 +62,8 @@
 
     public final static int IDX_COUNT = 6;
 
+    public final static int IDX_YEAR = 6;
+
```

Comment:
It's ```6``` on purpose. Year has the sixth index on the returned cursor.
  ```String query = "SELECT ROWID as _id, " + level
                + ", artist, album, title, cast(filename as TEXT), year FROM " + fromTable + " ";```

I could not figure out, what  ```IDX_COUNT``` is used for.